test/test_openapi.py

```diff
@@ -1,91 +1,112 @@
-from unittest.mock import MagicMock
+from collections import defaultdict
+from unittest import (
+    mock,
+    skip,
+)
 
+from azul.chalice import AzulChaliceApp
 from azul.openapi import (
-    get_app_specs,
-    openapi_spec,
     schema,
 )
 from azul_test_case import AzulTestCase
 
 
 class TestGetAppSpecs(AzulTestCase):
 
+    @skip('https://github.com/DataBiosphere/azul/issues/1450')
     def test_unannotated(self):
-        # TODO: Once all endpoints are documented, this test should assert failure
-        app = self._mock_app_object([lambda x: x])
-        get_app_specs(app)
+        app = self._mock_app_object()
+        self._mock_app_route(app, '/foo')
+        undocumented = app.routes.keys() - app.path_specs.keys()
+        self.assertFalse(undocumented)
```

Comment:
I prefer

```suggestion
        self.assertEqual(set(), undocumented)
```

My [earlier comment](https://github.com/DataBiosphere/azul/pull/1668#discussion_r405216252) on this was wrong but that doesn't mean there is nothing to be be addressed. You can always ask for clarification on Slack.